addon/components/frost-fixed-table.js

```diff
@@ -22,14 +22,17 @@ export default Component.extend({
   propTypes: {
     // options
     columns: PropTypes.arrayOf(ColumnPropType),
-    items: ItemsPropType
+    items: ItemsPropType,
```

Comment:
I personally wouldn't default them to empty arrays and would make them required. Isn't this component kind of useful without them? It'd be nice if I instantiate the component without one of these properties to see a warning land in my console. If I typo one of the properties the error would also land in my console.